src/Features/ERDDAP/Platform/Observations/Table/item.tsx

```diff
@@ -40,10 +40,12 @@ export const TableItem: React.SFC<TableItemProps> = ({
     data_type_list = data_type
   }
 
-  let data: PlatformDataset[] = []
+  let data: PlatformTimeSeries[] = []
 
   data_type_list.forEach((data_type) => {
-    platform.properties.readings.filter((ts) => data_type === ts.data_type.standard_name).forEach((ts) => data.push(ts))
+    platform.properties.readings
+      .filter((ts) => data_type === ts.data_type.standard_name && ts.depth < 2)
```

Comment:
Codacy found an issue: [Identifier 'data_type' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)